### ISSUE TITLE
Cross entropy loss mutliplied by training set size

### ISSUE THREAD

**Issue by anishmadan23** [senya-ashukha/variational-dropout-sparsifies-dnn#12]
I was running the Colab notebook (Pytorch) and had 2 questions:
1) Why is the loss going below 0? Is it due to the regularisation term added for the dropout? Also, I noticed that only when the loss went negative, I got a good compression ratio 
2) Why is cross-entropy loss multiplied by the training set size for each batch, since we usually divide the epoch loss by the dataset size?
I would really appreciate it if you could help me with my queries.
